avm/res/logic/integration-account/partner/main.bicep

```
@@ -0,0 +1,98 @@
+metadata name = 'Integration Account Partners'
+metadata description = 'This module deploys an Integration Account Partner.'
+
+@description('Required. The Name of the partner resource.')
+param name string
+
+@description('Optional. Resource location.')
+param location string = resourceGroup().location
+
+@description('Conditional. The name of the parent integration account. Required if the template is used in a standalone deployment.')
+param integrationAccountName string
+
+@description('Optional. B2B partner content settings.')
+param b2bPartnerContent b2bPartnerContentType?
```

Can use the resource-derived type
```suggestion
param b2bPartnerContent resourceInput<'Microsoft.Logic/integrationAccounts/partners@2019-05-01'>.properties.b2bPartnerContent?
```

---

Changed to derived type and matched api spec with resourceInput<'Microsoft.Logic/integrationAccounts/partners@2019-05-01'>.properties.content.b2b?